lib/subscriber.js

```diff
@@ -49,6 +49,11 @@ class SubscriptionContext {
     return this.pubsub.subscribe(topic, this.queue).then(() => this.queue)
   }
 
+  asyncIterator (topics) {
+    topics.forEach((topic) => this.pubsub.subscribe(topic, this.queue))
```

Comment:
`subscribe` returns a promise, we should use a `Promise.all()`

Reply:
If I return only the redable queue without having to do promise.all, it would be fine, isn't it? Anyway, processing is asynchronous.

Reply:
it's not the same thing, those promises needs to be awaited.

Reply:
If any error occurs, it has to stop, so promise.all seems to be correct as you say. I modified it...